android/samples/sample-gltf-viewer/src/main/java/com/google/android/filament/gltf/MainActivity.kt

```diff
@@ -74,6 +111,126 @@ class MainActivity : Activity() {
         modelViewer.view.bloomOptions = bloomOptions
     }
 
+    private fun bindPreview(cameraProvider: ProcessCameraProvider) {
+        val preview : Preview = Preview.Builder().build().also {
```

Comment:
I think you can leave off the `: Preview` here and `: CameraSelector` down below